scripts/module_builder.py

```diff
@@ -101,6 +101,8 @@ def write_source_file(source, config, strings):
 {{
     addToConfiguredModules<{strings[class_name]}, {strings[module_class_name]}>();
 }}
+
+template class {module_templ};
```

Comment:
Does this belong in this PR?

Reply:
Yes, if we want the Mac to build to tests successfully on Release. And it will be gone as soon as you rebase/merge from `develop` :P

Reply:
Ah, I see :)